srcpkgs/python3-requests-unixsocket/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'python3-requests-unixsocket'
+pkgname=python3-requests-unixsocket
+version=0.1.5
+revision=1
+noarch=yes
+wrksrc="requests-unixsocket-${version}"
+build_style=python3-module
+pycompile_module="requests_unixsocket"
+hostmakedepends="python3-setuptools python3-pbr"
+depends="python3-requests"
```

Comment:
might also be a good idea to python3-urllib3 here, as it is implicity brought by python3-requests